frontend/types/UserType.ts

```diff
@@ -16,4 +18,6 @@ export type UserProfileType = {
   name: string
   oneWord: string
   goal: string
+  github: string
+  x: string
```

Comment:
これってDBのテーブル定義的に

```suggestion
  github?: string
  x?: string
```

ですかね？

Reply:
変更しました！